addons/l10n_din5008_stock/report/din5008_stock_templates.xml

```diff
@@ -8,7 +8,6 @@
                         <span class="fw-bold" t-if="o.picking_type_id.code == 'incoming'">Vendor Address:</span>
                         <span class="fw-bold" t-if="o.picking_type_id.code == 'internal'">Warehouse Address:</span>
                         <span class="fw-bold" t-if="o.picking_type_id.code == 'outgoing' and o.move_ids_without_package and o.move_ids_without_package[0].partner_id and o.move_ids_without_package[0].partner_id.id != o.partner_id.id">Customer Address:</span>
-                        <address t-esc="o.partner_id" t-options='{"widget": "contact", "fields": ["address", "name", "phone"], "no_marker": True}'/>
```

Comment:
I don't understand, we remove the address, but keep the label in the 3 cases above? and in those case there will be a label but the address will not be shown?